include/RAJA/policy/cuda/forall.hpp

```diff
@@ -205,10 +203,8 @@ RAJA_INLINE void forall(cuda_exec<BLOCK_SIZE, Async>,
   auto final_end = std::end(iter);
   auto total_len = std::distance(first_begin, final_end);
   auto max_step_size = (getCudaMemblockUsedCount() > 0)
-                                 ? BLOCK_SIZE * RAJA_CUDA_MAX_NUM_BLOCKS
-                                 : total_len;
-
-  RAJA_FT_BEGIN;
```

Comment:
Or you could remove the macro calls from the traversal methods, but leave the header file that defines those macros in place for reference.